routers/repo/view.go

```diff
@@ -333,7 +333,25 @@ func renderDirectory(ctx *context.Context, treeLink string) {
 	// Show latest commit info of repository in table header,
 	// or of directory if not in root directory.
 	ctx.Data["LatestCommit"] = latestCommit
-	ctx.Data["LatestCommitVerification"] = models.ParseCommitWithSignature(latestCommit)
+	verification := models.ParseCommitWithSignature(latestCommit)
+
+	ctx.Data["LatestCommitVerification"] = verification
+	verification.TrustStatus = "trusted"
+	if verification.Verified && verification.SigningUser.ID != 0 {
```

Comment:
This logic is more or less repeated three times. Perhaps it would be better to move this to its own function?

Reply:
Done